### PR TITLE
[PBW-3792] resize recalculates scrubber bar width to place playhead a…

### PR DESCRIPTION
**Pull request by ksunika**
…t the correct point

### CODE REVIEW

File: js/components/scrubberBar.js [hunk 1]
```diff
@@ -37,6 +37,27 @@ var ScrubberBar = React.createClass({
     this.scrubberBarWidth = this.getDOMNode().querySelector(".scrubberBar").clientWidth;
     this.scrubberBarHeight = this.getDOMNode().querySelector(".scrubberBar").clientHeight;
     this.playheadWidth = this.getDOMNode().querySelector(".playhead").clientWidth;
+
+    // Make sure component resize correctly after switch to fullscreen/inline screen
+    window.addEventListener('resize', this.handleResize);
+    window.addEventListener('webkitfullscreenchange', this.handleResize);
```
Comment (mistermendez) [inline js/components/scrubberBar.js]: There's a mixin for that - https://github.com/ooyala/html5-skin/blob/master/js/mixins/resizeMixin.js

Comment (mistermendez) [inline js/components/scrubberBar.js]: Example use - https://github.com/ooyala/html5-skin/blob/master/js/views/playingScreen.js#L13

Comment (ksunika) [inline js/components/scrubberBar.js]: cool, I'll use it then!

**Comment by ksunika** [comment]
@mistermendez, what about now?

**Comment by mistermendez** [comment]
:+1:

**Pull request merged**